content/docs/0.7.x/operate/k8s_support/index.md

```diff
@@ -21,15 +21,15 @@ Keptn versions are expressed as `x.y.z`, where `x` is the major version, `y` is
 
 | Keptn Version /<br>Installation                         | Kubernetes        | AKS               | EKS                   | GKE               | OpenShift   | K3s         | Minikube                                         | MicroK8s       | Minishift                 |
 |---------------------------------------------------------|:-----------------:|:-----------------:|:---------------------:|:-----------------:|:-----------:|:-----------:|:------------------------------------------------:|:--------------:|:--------------------------|
-| **0.7.x** / <br>Control & Execution plane<br>*see: (3)* | 1.19 - 1.14 | 1.16 - 1.15<br>*see: (1)* | 1.16 - 1.14         | 1.17 - 1.14       | 4, 3.11     | 1.19 - 1.16 | 1.10.1<br>(K8s:1.18.2) -<br> 1.3.1<br>(K8s:1.15) | -              | 1.34.2<br>(K8s: 1.11)     |
-| **0.7.x** / <br>Control plane                           | 1.19 - 1.14 | 1.16 - 1.15<br>*see: (1)* | 1.16 - 1.14         | 1.17 - 1.14       | 4, 3.11     | 1.19 - 1.16 | 1.10.1<br>(K8s:1.18.2) -<br> 1.3.1<br>(K8s:1.15) | 1.19 - 1.16    | 1.34.2<br>(K8s: 1.11)     |
+| **0.7.x** / <br>Control & Execution plane<br>*see: (3)* | 1.19 - 1.14 | 1.18 - 1.15<br>*see: (1)* | 1.18 - 1.14         | 1.18 - 1.14       | 4, 3.11     | 1.19 - 1.16 | 1.10.1<br>(K8s:1.18.2) -<br> 1.3.1<br>(K8s:1.15) | -              | 1.34.2<br>(K8s: 1.11)     |
+| **0.7.x** / <br>Control plane                           | 1.19 - 1.14 | 1.18 - 1.15<br>*see: (1)* | 1.18 - 1.14         | 1.18 - 1.14       | 4, 3.11     | 1.19 - 1.16 | 1.10.1<br>(K8s:1.18.2) -<br> 1.3.1<br>(K8s:1.15) | 1.19 - 1.16    | 1.34.2<br>(K8s: 1.11)     |
```

Comment:
Not sure why this file was updated, but okay

Reply:
according to the Slack channel, folks run 0.7.3 successfully on 1.18. I would update it accordingly.